plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/inspector.tsx

```diff
@@ -27,7 +27,7 @@ import {
 import { sortOrderOptions } from './constants';
 import { BlockAttributes, EditProps } from './types';
 import { getAttributeFromId } from './utils';
-import { getInnerBlockByName } from '../../utils';
+import { getInnerBlockByName } from '../../utils/get-inner-block-by-name';
```

Comment:
I removed the index file of exports, which is not ideal when mixing some utils used both frontend and backend.